modules/openapi-generator/src/main/resources/cpp-pistache-server/model-header.mustache

```diff
@@ -50,13 +62,20 @@ public:
     void unset{{name}}();
     {{/required}}
     {{/vars}}
+    {{#isEnum}}
+    {{classname}}::e{{classname}} getValue() const;
+    void setValue({{classname}}::e{{classname}} value);
+    {{/isEnum}}
 
 protected:
     {{#vars}}
     {{{dataType}}} m_{{name}};
     {{^required}}
     bool m_{{name}}IsSet;{{/required}}
     {{/vars}}
+    {{#isEnum}}
```

Comment:
does this not belong in the vars section of the template?

Reply:
Please see my comment [here](https://github.com/OpenAPITools/openapi-generator/pull/1366#discussion_r230570943)